pages/code-of-conduct.md

```diff
@@ -0,0 +1,31 @@
+BuildersLeaders Code of Conduct
```

Comment:
@vire I will need a little guidance how to add this to the landing page as ref :)

Reply:
What I would do for now - create a `<a href="full-url" >CODE OF CONDUCT</a>` to the CoC here on github and add logic to render the .md -> http://geekhmer.github.io/blog/2018/03/29/import-markdown-files-and-serve-its-content-in-next-dot-js/

Reply:
After several unsuccessful attempts I give up and added simple ref to github (which will render it for us) :)

Should work for now, let's improve in the future